sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Receipt.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+using Azure.AI.FormRecognizer.Models;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    /// <summary>
+    /// Represents a receipt recognized from the input document and provides strongly-typed members
+    /// for accessing common fields present in recognized receipts.
+    /// </summary>
+    public class Receipt
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Receipt"/> class.
+        /// </summary>
+        /// <param name="recognizedForm">The form from which information will be extracted to build this receipt.</param>
+        public Receipt(RecognizedForm recognizedForm)
+        {
+            ReceiptType = ConvertStringField("ReceiptType", recognizedForm.Fields);
+            MerchantAddress = ConvertStringField("MerchantAddress", recognizedForm.Fields);
+            MerchantName = ConvertStringField("MerchantName", recognizedForm.Fields);
+            MerchantPhoneNumber = ConvertPhoneNumberField("MerchantPhoneNumber", recognizedForm.Fields);
+            Subtotal = ConvertFloatField("Subtotal", recognizedForm.Fields);
+            Tax = ConvertFloatField("Tax", recognizedForm.Fields);
+            Tip = ConvertFloatField("Tip", recognizedForm.Fields);
+            Total = ConvertFloatField("Total", recognizedForm.Fields);
+            TransactionDate = ConvertDateField("TransactionDate", recognizedForm.Fields);
+            TransactionTime = ConvertTimeField("TransactionTime", recognizedForm.Fields);
```

Comment:
Yes, I agree with that.  It might even be nice to link to the documented receipt fields here or elsewhere in the sample to let readers know why this set of fields was chosen.